port/zos390/omrsl.c

```diff
@@ -49,8 +53,14 @@ omrsl_open_shared_library(struct OMRPortLibrary *portLibrary, char *name, uintpt
 	char *openName = name;
 	char mangledName[EsMaxPath + 1];
```

Comment:
unintialized variables were initialized in `omrsl_lookup_name`. we can do the same for `handle = NULL`, `mangledName = {0}` and `errBuf = {0}`.

Reply:
Fixed in https://github.com/eclipse/omr/pull/5834/commits/6fcac92aef3a9768ca39f66030366bad4d7f7fff